src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -65,7 +66,9 @@ public void periodic() {
                             inputs[i].pose.get()[4],
                             inputs[i].pose.get()[5],
                             inputs[i].pose.get()[6]))
-                    .rotateBy(new Rotation3d(0, 0, Math.PI));
+                    .unaryMinus()
```

Comment:
can't you just rotate by `Rotation3d(0, 0, -Math.PI)` instead of using two `unaryMinus()`'s?

Reply:
I don't think so, 3D rotation isn't commutative nor associative.